<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemCharacteristic</class>
 <widget class="QDialog" name="DialogSystemCharacteristic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set characteristic...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Buff</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" rowspan="2" colspan="2">
          <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,1">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonBuff">
             <property name="text">
              <string>Buff:</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QComboBox" name="comboBoxIncreaseDecrease">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="3" colspan="3">
            <widget class="QWidget" name="widget" native="true">
             <property name="minimumSize">
              <size>
               <width>542</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>542</width>
               <height>27</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
                <item>
                 <widget class="QComboBox" name="comboBoxIncreaseDecreaseKind">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>27</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>27</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1,1,1,1,1,1">
                  <item>
                   <widget class="PanelPrimitiveValue" name="panelPrimitiveValueStatValue" native="true"/>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBoxSkillCostAll">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="PanelPrimitiveValue" name="panelPrimitiveValueElementRes" native="true"/>
                  </item>
                  <item>
                   <widget class="PanelPrimitiveValue" name="panelPrimitiveValueStatusRes" native="true"/>
                  </item>
                  <item>
                   <widget class="PanelPrimitiveValue" name="panelPrimitiveValueCurrencyGain" native="true"/>
                  </item>
                  <item>
                   <widget class="PanelPrimitiveValue" name="panelPrimitiveValueSkillCost" native="true"/>
                  </item>
                  <item>
                   <widget class="WidgetVariable" name="widgetVariable" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>250</width>
                      <height>27</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>250</width>
                      <height>27</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_11">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>0</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>158</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>24</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBoxOperation">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>*</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValue" native="true">
             <property name="minimumSize">
              <size>
               <width>290</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>290</width>
               <height>27</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QComboBox" name="comboBoxUnit">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>%</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="5" colspan="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>278</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="7">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>707</width>
               <height>35</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="panelPrimitiveValueVariable" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Character specific</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" colspan="2">
            <widget class="QRadioButton" name="radioButtonEquip">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Equip:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="2">
            <widget class="QComboBox" name="comboBoxEquipAllowForbid">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Allow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Forbid</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="4" colspan="2">
            <widget class="QComboBox" name="comboBoxEquipWeaponArmor">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Weapon</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Armor</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="6" colspan="5">
            <widget class="QWidget" name="widget_3" native="true">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
                <item>
                 <widget class="PanelPrimitiveValue" name="panelPrimitiveValueWeaponKindID" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="PanelPrimitiveValue" name="panelPrimitiveValueArmorKindID" native="true"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="11">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>178</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="4">
            <widget class="QRadioButton" name="radioButtonEquipmentChange">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Equipment change:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QComboBox" name="comboBoxEquipmentChange">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Allow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Forbid</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="5" colspan="5">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueEquipmentChangeID" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="10" colspan="2">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>208</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="0" colspan="4">
            <widget class="QRadioButton" name="radioButtonBeginEquipment">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Begin equipment:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4" colspan="4">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueBeginEquipmentID" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="8">
            <widget class="QLabel" name="labelWith">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string> with </string>
             </property>
            </widget>
           </item>
           <item row="2" column="9" colspan="3">
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>238</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QComboBox" name="comboBoxBeginWeaponArmor">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>Weapon</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Armor</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="3" colspan="4">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueBeginWeaponArmorID" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="7" colspan="5">
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>298</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0" colspan="12">
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>668</width>
               <height>28</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Other</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,1">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonScript">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Script:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="PanelPrimitiveValue" name="panelPrimitiveValueScript" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>27</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0" colspan="3">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>378</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetVariable</class>
   <extends>QWidget</extends>
   <header>widgetvariable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemCharacteristic</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemCharacteristic</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
